### ISSUE TITLE
Construct2 Video doesnt play on IOS11 (probably playinline issue) 

### ISSUE THREAD

**Issue by pixcubeanim** [Scirra/Construct-3-bugs#895]
Ref: https://www.scirra.com/forum/viewtopic.php?t=196484

Repro: https://www.dropbox.com/s/4215c1d0jmqp8sc/video_test.zip?dl=0

inside the zip file you can find a construct2/construct3 file and a ipa, builded with construct2 for cordova (it is important because this issue depends on ios11 / mobile application)

## Steps to reproduce

1. Open the app and press the button
2. A video should appear but it doesnt

## Expected result

The video should play inside the canvas (not fullscreen)

## Affected devices
On Construct2: Each ios device with version 11
On Construct3: Each ios device, it doesnt matter which version

## System details

There are no crash reports available. The video canvas seems to be not recognized. The device specifications are irelevant, this application (testfile) should run on each ios device.

## Notes:
Iam not sure, but if youre building for cordova, you can customize the config.xml file / construct2.runtime file. If youre deleting the "playinline" path, the app will play the videos in fullscreen mode (very bad but at the end it recognizes a video - which is a small success)

So i guess ios or cordova or construct has a problem with this... iam not an expert in this matter, so i need help - its very important. 

Thank you very much!

**Comment by AshleyScirra** [comment]
Have you tested iPhone and iPad separately? Does it reproduce on both or have you only tested one? IIRC they treat video playback differently so the type of device you are using is an important factor.

**Comment by pixcubeanim** [comment]
Hey Ashley, 

yes basically i tested it on iphone and ipad.... the problem currently still appears. On Ipad: <playsinline> was working until version 11. On iphone ive made a version where fullscreen mode couldnt be avoided. But basically the problem i have is that there is no video showing on ios11 at all.

**Comment by AshleyScirra** [comment]
OK, it looks like our HTTP server for video playback hack doesn't work in iOS 11. I found another workaround which looks like it works; let's check this again when r61 is out and make sure it's working as expected.

**Comment by AshleyScirra** [comment]
Construct 3 r61 is out, can you check again with that?

**Comment by AshleyScirra** [comment]
Been about 2 weeks with no response so closing.

**Comment by pixcubeanim** [comment]
@AshleyScirra 
Sorry i had vacation. The problem is still present. I will test it these days again.

**Comment by AshleyScirra** [comment]
I had previously verified it works so if you still have trouble please file a new issue following all the guidelines.
